tests/containers/seccomp.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: seccomp
+# Summary: Test seccomp in docker & podman
+# Maintainer: QE-C team <qa-c@suse.de>
+
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils;
+use containers::common;
+
+sub run {
+    my ($self, $args) = @_;
+    my $runtime = $args->{runtime};
+
+    select_serial_terminal;
+    my $engine = $self->containers_factory($runtime);
+
+    assert_script_run "grep SECCOMP /boot/config-\$(uname -r)";
+    assert_script_run "$runtime info | grep -i seccomp";
+
+    my $image = "registry.opensuse.org/opensuse/bci/bci-busybox:latest";
+    my $policy = "policy.json";
+
+    # Get default policy
+    if ($runtime eq "docker") {
+        script_retry("curl -sLo $policy https://github.com/moby/moby/raw/master/profiles/seccomp/default.json", retry => 5, delay => 60, timeout => 300);
+    } elsif ($runtime eq "podman") {
+        assert_script_run "cp \$(podman info -f '{{ .Host.Security.SECCOMPProfilePath }}' $policy";
+    }
+
+    # Run ls command
+    assert_script_run "$runtime run --rm --security-opt seccomp=$policy $image ls >/dev/null";
+    # Remove syscalls needed to get directory entries
+    assert_script_run "sed -i -e 's/\"getdents\",//' -e 's/\"getdents64\",//' $policy";
+    assert_script_run "! $runtime run --rm --security-opt seccomp=$policy $image ls >/dev/null";
+
+    # Try to bind-mount /proc to /home
+    assert_script_run "$runtime run --rm --cap-add SYS_ADMIN --security-opt seccomp=unconfined $image mount --bind /proc /home";
+
+    $engine->cleanup_system_host();
```

Comment:
I would do this in the `post_run_hook` and `post_fail_hook`, otherwise the system remains "unclean" on test failure.

Reply:
Fixed.